### PR TITLE
feat: Add flow charts for eli3 and eyfs

### PR DESCRIPTION
**Pull request by kstathou**
Adding a flowchart for each prototype to illustrate how they work.

### CODE REVIEW

**Comment by kstathou** [comment]
@beingkk, it would be great to add one for your whatsapp bot too in the future!

File: README.md [hunk 1]
```diff
@@ -3,12 +3,27 @@
 Collection of generative AI prototypes, mainly using LLMs.
 
 - [Generative AI prototypes](#generative-ai-prototypes)
+  - [Prototypes](#prototypes)
+    - [Explain me like I am a 3 year old](#explain-me-like-i-am-a-3-year-old)
```
Comment (beingkk) [inline README.md]: I think it should be "explain it to me like.." or "explain like.."
```suggestion
    - [Explain like I am a 3 year old](#explain-me-like-i-am-a-3-year-old)
```

File: README.md [hunk 2]
```diff
@@ -3,12 +3,27 @@
...
+    - [Suggesting personalised early-years activities](#suggesting-personalised-early-years-activities)
   - [Setup](#setup)
     - [Generic setup for working with `pyenv` and `poetry`](#generic-setup-for-working-with-pyenv-and-poetry)
     - [How to install this project](#how-to-install-this-project)
   - [Launch the streamlit app](#launch-the-streamlit-app)
   - [TODO](#todo)
 
+## Prototypes
+
+### Explain me like I am a 3 year old
```
Comment (beingkk) [inline README.md]:
```suggestion
### Explain like I am a 3 year old
```

File: charts/eyfs.png
Comment (beingkk) [inline charts/eyfs.png:1]: I think temperature in the flowchart should be "0....1" rather than "0....2"

**Review by beingkk — CHANGES_REQUESTED** [review]
Thank you @kstathou, great idea to add flowcharts! Did you use Figma for these?

I noticed a couple small adjustments, but otherwise looks great.

File: charts/eyfs.png
Comment (kstathou) [reply charts/eyfs.png:1]: I think it used to be 0-1 but it's 0-2 now! https://platform.openai.com/docs/api-reference/chat/create#chat/create-temperature

**Comment by kstathou** [comment]
> Thank you @kstathou, great idea to add flowcharts! Did you use Figma for these?
> 
> I noticed a couple small adjustments, but otherwise looks great.

I've used [excalidraw](https://excalidraw.com/) for them which is open source, free to use and works locally too!

File: charts/eyfs.png
Comment (beingkk) [reply charts/eyfs.png:1]: Oh wow, double the creativeness!

**Review by beingkk — APPROVED** [review]
Looks great, thank you!

**Pull request merged**